### ISSUE TITLE
Classify data and test functions

### ISSUE THREAD

**Issue by dflorit** [Andy-Daniela/Leaves-Classifier#26]
build function that:
1. Reads data from Kaggle's training data table using: read_kaggle_training_table()
2. Splits data into training set and testing set using: split_data()
3. Creates an SVM classifier
4. trains the classifier with the feature_vectors from the table
5. predicts labels for testing set
6. evaluates performance

7. Creates an NN classifier
8-10. Repeats steps 4-6 but using the NN classifier

11. compare performances
